runtime/compiler/trj9/env/J9SharedCache.cpp

```diff
@@ -455,7 +455,8 @@ TR_J9SharedCache::rememberClass(J9Class *clazz, bool create)
       return NULL;
       }
 
-   char key[17]; // longest possible key length is way less than 16 digits
+   const uint32_t bufferLength = 17; // longest possible key length is way less than 16 digits
+   char key[bufferLength];
```

Comment:
Hm should I change this back? I had modified this line when I thought I had to change the number of arguments to `createClassKey` but as it turns out, I didn't need to. I guess I forgot to revert this.